### PR TITLE
Add timeline of changes element

### PR DESCRIPTION
**Pull request by DilwoarH**
## What
Add a timeline of recent changes to the landing page to communicate to users what rules are changing or have recently changed, and what they need to do differently from what date.

## Why
We know that users are struggling to find out what has recently changed, and this impacts how likely they are to follow the rules, and how confident they feel that the landing page is up to date and relevant.

We hope that this timeline element will help users:

- understand what's changed
- feel confident that the information is up to date
- ultimately comply with new rules and help prevent the spread of covid

https://trello.com/c/XUyYJ8lt/813-24-sept-landing-page-add-timeline-of-changes-element

### CODE REVIEW

**Comment by DilwoarH** [comment]
Will need content sign off by @helengds before we merge

File: test/presenters/coronavirus_landing_page_presenter_test.rb
Comment (leenagupte) [inline test/presenters/coronavirus_landing_page_presenter_test.rb:7]: Also not a blocker, but this line has also become very long. It's probably at the point where each list item should be on a separate line.

File: app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb
Comment (leenagupte) [reply app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb:38]: Have you test how this affects the accordions on the hub pages?

Live hub page: 
Screenshot 2020-09-22 at 17 07 15

Hub page with local changes:

Screenshot 2020-09-22 at 17 07 58

Is that extra black HR supposed to be on the hub pages as well?

**Review by leenagupte — COMMENTED** [review]
In general it looks good. There are a couple of comments, most aren't blockers.

The big concern I have is about the new black line on the hub pages (see screenshots in inline comments)

File: app/presenters/coronavirus_landing_page_presenter.rb
Comment (DilwoarH) [reply app/presenters/coronavirus_landing_page_presenter.rb]: Done

File: app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb]: Thanks for spotting this, done ✅

File: test/presenters/coronavirus_landing_page_presenter_test.rb
Comment (DilwoarH) [reply test/presenters/coronavirus_landing_page_presenter_test.rb]: ✅

**Comment by DilwoarH** [comment]
@leenagupte - thanks for the review - I've addressed the things you raised

**Review by leenagupte — APPROVED** [review]
Woop! Looks great! 🎉 

I've tested the changes locally, and the landing page and hub pages look as they should.

I think you'll need to rebase against master to get the publishing e2e tests to pass.

**Comment by DilwoarH** [comment]
Reminder to not merge this until @helengds has signed off

**Pull request merged**
